lib/types/form-types.ts

```diff
@@ -142,6 +142,7 @@ export interface DeliveryOption {
 }
 
 export type SecurityAttribute = "Unclassified" | "Protected A" | "Protected B";
+export type FormPurpose = "" | "admin" | "nonAdmin";
```

Comment:
Can we update or replace the current enum ?

```
export enum PurposeOption {
  none = "",
  admin = "admin",
  nonAdmin = "nonAdmin",
}
```

Reply:
This is okay as just minor concerns around having multiple types for the PurposeOption and needing to update in multiple places (possibly missing one or the other) if the types ever change.

 

We could use as const vs the enum:

<img width="1026" alt="Screenshot 2024-07-03 at 8 10 10 AM" src="https://github.com/cds-snc/platform-forms-client/assets/62242/2cb9a438-f9de-4a58-94af-12b9405a31fc">


https://youtu.be/jjMbPt_H3RQ?t=320

---

We can leave as is for now.